docs/contributions/1_local_setup.md

```diff
@@ -202,60 +186,26 @@ copy sample.env .env
 
 <!-- tabs:end -->
 
-All the secret keys are left blank in `sample.env` but you can request access to [this](https://docs.google.com/document/d/1D4CJ9l6lnR39YYPUvw_HbeUVXNR-tAbNF6eT89oxEuk) google doc to see those
-
-The keys in the `.env` file are _not_ required to be changed to run the app locally. You can leave the default values copied over from `sample.env` as-is.
-
-<details markdown="1">
-<summary>
-If you want to login, follow these steps to retrieve the ``KEYCLOAK_SECRET`` value
-</summary>
-1. In the [#_software-dev slack channel](https://starvoting.slack.com/archives/C01EBAT283H), ask for keycloak admin access, and we can set you up with an account (assuming we trust you of course ;) )
-1. Navigate to https://keycloak.6j0.org/ and click Administration Console
-1. Login w/ your admin credentials (this is using the master realm, so it's different from your star.vote credentials)
-1. Navigate to Clients > star_vote_web > credentials
-1. Copy the secret and paste it to your .env file (DONT HIT "Regenerate Secret", this will invalidate the previous secret and break login for everyone)
-
-![](../images/keycloak_secret.png)
-
-</details>
-
-#### Step 2: Install dependencies
-
-Install the frontend dependencies
-
-```
-cd frontend
-npm install
-```
-
-Install the backend dependencies
-
-```bash
-cd backend
-npm install
-```
-
-#### Step 3: Start the star.vote client application and API server
+#### Step 3: Install dependencies and start the star.vote client application and API server
 
 You can now start up the API server and the client applications.
 
 Launch backend 
 
 ```bash
 cd backend
+npm install
 npm run dev
 ```
 
-Launch frontend
+Launch frontend. If you want to just run the web client without setting up the server you can change the proxy under frontend/package.json to "proxy": "dev.star.vote". 
```

Comment:
Were you able to get the proxy to work? 

I used to use this trick when we were on heroku, but I haven't had success with the dev.star.vote endpoint yet

Reply:
I didn't try it - assumed it was still working 😬 
I was able to get it to look at other local port here.